cloudferry_devlab/cloudferry_devlab/tests/testcases/test_keystone_migration.py

```diff
@@ -34,24 +36,43 @@ def test_migrate_keystone_users(self, param):
         :param name: user's name
         :param description: user's description
         :param enabled: is user enabled"""
-        self.validate_resource_parameter_in_dst(self.src_users, self.dst_users,
+        filtered_src_users = [user for user in self.src_users
+                              if getattr(user, "name") !=
+                              config.case_sensitivity_test_user]
+        self.validate_resource_parameter_in_dst(filtered_src_users,
+                                                self.dst_users,
                                                 resource_name='user',
                                                 parameter=param)
 
-    @attr(migrated_tenant=['admin', 'tenant1', 'tenant2'])
+    @attr(migrated_tenant=['admin', 'tenant1', 'tenant2', 'tenant6'])
     def test_migrate_keystone_user_tenant_roles(self):
         """Validate user's tenant roles were migrated with correct name."""
-        src_user_names = [user.name for user in self.filter_users()]
+        tenant = base.get_nosetest_cmd_attribute_val('migrated_tenant')
```

Comment:
Using environment variable in test does not seem to be correct: given that generated data is part of test, I don't see any reason why this value needs to be passed as external dependency. Please rework to make it independent of environment variables.